packages/osd-plugin-generator/src/integration_tests/generate_plugin.test.ts

```diff
@@ -32,13 +32,17 @@ import Path from 'path';
 
 import del from 'del';
 import execa from 'execa';
-import { REPO_ROOT } from '@osd/utils';
-import { createAbsolutePathSerializer } from '@osd/dev-utils';
+import { REPO_ROOT, standardize, createAbsolutePathSerializer } from '@osd/dev-utils';
 import globby from 'globby';
 
-const GENERATED_DIR = Path.resolve(REPO_ROOT, `plugins`);
+// Has to be a posix reference because it is used to generate glob patterns
+const GENERATED_DIR = standardize(Path.resolve(REPO_ROOT, `plugins`), true);
 
-expect.addSnapshotSerializer(createAbsolutePathSerializer());
+expect.addSnapshotSerializer(
+  createAbsolutePathSerializer(
+    process?.platform === 'win32' ? standardize(REPO_ROOT, true) : REPO_ROOT
```

Comment:
same question. can process be undefined?

Reply:
Same :D